src/ImageSharp/Formats/Tiff/TiffMetadata.cs

```diff
@@ -26,6 +26,14 @@ public TiffMetadata()
         /// </summary>
         public ByteOrder ByteOrder { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether this instance is BigTiff format.
+        /// </summary>
+        /// <value>
+        ///   <c>true</c> if this instance BigTiff format; otherwise, <c>false</c>.
+        /// </value>
+        public bool IsBigTiff { get; set; }
```

Comment:
I don't think we should do it like this. I'm always wary of booleans as they limit future expansion. An enum would allow us to expand into future things like GeoTIFF

```c#
public enum TiffFormatType
{
    Default,
    BigTIFF
}
```

Reply:
yeah, done